charts/couchdb/templates/pv.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
I don't think using PV here is the best way to use a persistent store. As using PV, need to config the volume source/type and detail configs(e.g. NFS path). It's better to use a dynamic provisioner to do so, which means except for a PVC definition, we also need a storage class definition.